fleet/lib/rook-ceph-conf/charts/manke/templates/cephnfs-rsphome.yaml

```diff
@@ -0,0 +1,85 @@
+---
+apiVersion: ceph.rook.io/v1
+kind: CephFilesystem
+metadata:
+  name: rsphome
+  namespace: rook-ceph
+spec:
+  metadataPool:
+    failureDomain: host
+    replicated:
+      size: 3
+    quotas:
+      maxSize: 10Gi
+  dataPools:
+    - name: default
+      failureDomain: host
+      replicated:
+        size: 3
+      quotas:
+        maxSize: 250Gi
+    - name: ec
+      failureDomain: host
+      erasureCoded:
+        dataChunks: 6
+        codingChunks: 3
```

Comment:
This seems large for a 10 node cluster with fault tolerance of 2.  6+2 is probably safer. I wonder if 5+3 is a good idea?